common/v2/services/EthService/nodes/web3/node.ts

```diff
@@ -78,7 +78,9 @@ export async function setupWeb3Node() {
   const { ethereum } = window as any;
```

Comment:
- To clean up as much as we can before going out of beta, can you delete  the uncalled methods `ensureWeb3NodeStillAvailable` & `isWeb3NodeAvailable` please?
- ```
      if (ethereum) {
        // Overwrite the legacy Web3 with the newer version.	
	       if ((window as any).Web3) {
           (window as any).web3 = new (window as any).Web3(ethereum);
         }
         //....
  ```
Nested conditionals start to smell and the repeated access to window as well. How about we use `HAS_WEB3_PROVIDER` or siblings for the control flow?

Reply:
Removed the methods. What do you prefer for the nested conditionals? The whole thing is quite hacky either way.

Reply:
True. Let's leave it for now then.